devel/external/test_atm_psf/psf_movie.py

```diff
@@ -0,0 +1,110 @@
+import matplotlib
```

Comment:
Very minor point about the location of this.  I think devel/external should be reserved for cases where we are using some non-GalSim code to do a calculations.  e.g. the original Shera code, the original SBProfile code, Mathematica, Maple, IDL, calculations on the COSMOS data, calculations using phosim output, etc.  

Admittedly, not everything in there now really fits this, but I think this would be better placed in a subdirectory of devel/modules/.  e.g. devel/modules/phase_psf.  And probably ought to move the ArScreens stuff into this dir as well.